cmd/gossamer/config_test.go

```diff
@@ -513,6 +513,44 @@ func TestUpdateConfigFromGenesisJSON(t *testing.T) {
 	require.Equal(t, expected, cfg)
 }
 
+// TestUpdateConfigFromGenesisJSON_Default tests updateDotConfigFromGenesisJSON
+// using the default genesis path if no genesis path is provided (ie, an empty
+// genesis value provided in the toml configuration file or with --genesis "")
+func TestUpdateConfigFromGenesisJSON_Default(t *testing.T) {
```

Comment:
what happens if the node is loaded with a non default genesis then ran with a config without genesis?

Reply:
This just tests the added check for the `updateConfigFromGenesisJSON` method. When the method is used with a toml configuration file that has an empty genesis init value or an empty genesis value is passed using the --genesis, it should use the default genesis method.

If the configuration at the time `updateConfigFromGenesisJSON` is called includes an empty value for the genesis init value, it will use the default value, which is the same value being used if we run a node with no flags (ie, `./bin/gossamer init`).

I am not sure I completely understand your question but if you run `gossamer init --config test.toml --genesis test.json` and the config file has an empty value for the genesis init value, it will use the genesis init value provided within the flag option.